New article to understand data quality feature (#430)
akeneo/pim-helpcenter: content/md/products-super-power/understand-data-quality.md

---
id: understand-data-quality
themes: products-super-power, first-steps
title: Understand **data quality**
popular: false
ee-only: true
related:
---

# About data quality
What is this **red, orange or green A, B, C, D, E displayed on my product edit form**? What are the **enrichment** and **consistency** grades?

**This is data quality insights!**

> Because rich, coherent and consistent product data is important to you, we introduced the concept of "data quality" in our PIM.

Reviewer: ```suggestion
Because rich, coherent and consistent product data is important for delivering great product experiences, we introduced the concept of "data quality" in our PIM.
```

> This new capability helps you measure your data quality on two axes: **enrichment** and **consistency**.

Reviewer: ```suggestion
This new capability helps you measure your data quality along two axes: **enrichment** and **consistency**.
```

> Akeneo's data quality is an information about your product data quality: it helps you measure your data quality on two axis: **enrichment** and **consistency**.

Reviewer 2: ```suggestion
This new capability helps you measure your data quality on two axes: **enrichment** and **consistency**.
```

> Each of these two axes has a score from A to E. The scores can be seen on several screens of the PIM.```

Author: ```suggestion
Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.
```

> Each of these two axis has a grade ranging from A to E. The grades can be seen on several screens of the PIM.

Reviewer 2: ```suggestion
Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.```

Author: ```suggestion
Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.
```

Author: ```suggestion
Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.
```

![Image data quality badges](../img/data-quality-grades.png)

# How is enrichment calculated?

> The **enrichment grade** is calculated on the percentage of attributes filled in your product edit form (whether the attributes are required for completeness or not).

Reviewer: ```suggestion
The **enrichment grade** is calculated as the percentage of attributes with values in your product edit form (whether the attributes are required for completeness or not).
```

> Checking the `data quality insights` panel in the product edit form allows you to quickly see which attributes need to be filled to improve your data quality grade for enrichment.

Reviewer: ```suggestion
Checking the `data quality insights` panel in the product edit form allows you to quickly see which attributes need to be filled in to improve your data quality grade for enrichment.
```

::: tips

> From the `data quality insights` panel, by clicking on the "Display all missing attributes", you can easily access a product edit form filtered on all the attributes that need to be filled.

Reviewer: ```suggestion
From the `data quality insights` panel, by clicking on the "Display all missing attributes", you can easily access a product edit form filtered on all the attributes that need to be filled in.
```

:::

# How is consistency calculated?

> The **consistency grade** is calculated on several criteria. The mix of the following criteria help the PIM determine the grade for consistency:

Reviewer: ```suggestion
The **consistency grade** is calculated on several criteria. The mix of the following criteria helps the PIM determine the grade for consistency:
```

- Spell check: text and text areas are checked for spelling mistakes.

> - Title formatting: the attribute as main title is checked for optimised writing.

Reviewer 2: ```suggestion
- Title formatting: the attribute used as the main product title is checked for optimized writing.
```

- Upper cases formatting: the PIM checks whether text area attributes are correctly formatted for upper cases.
- Lower cases formatting: the PIM checks whether text area attributes are correctly formatted for lower cases.

::: info
Spell check is available for English, French, German and Spanish locales.

> Spell check is usable with text and simple text area attributes.

Reviewer 2: "simple text are" means without the WYSIWYG enabled right?

Author: yes

Reviewer: on lines 35 and 36, normally one uses "uppercase" and "lowercase" as one word when it is an adjective or a noun -- but two words isn't wrong. And, they are not normally plural, so it should be "uppercase formatting" or "upper case formatting."

Author: Thnks @jmevans00 I asked Camille about this and she said both could be used as well. Which one do you suggest we use to be most universal? I'll go with your pick ;-)

Reviewer: my suggestion is "uppercase formatting"

:::

::: info

> Title formatting suggestion is available for English locales.

Reviewer: ```suggestion
Title formatting suggestions are available for English locales.
```

Reviewer: or, "The title formatting suggestion is available for English locales."

:::

::: tips
From the `data quality insights` panel, by clicking on the "Display all improvable attributes", you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
:::

::: success

> To take full advantage of the title formatting functionality, set the right attribute to be used as the main product title

Reviewer 3: Not sure that "set the right attribute to be used as the main product title" is the right terminology.

> :::

Reviewer 3: Not sure that "set the right attribute to be used as the main product title" is the right terminology.

Author: Thanks @nadavruppin I changed it

> To fully take advantage of the Title formatting criteria, set your product title as attribute as main title

Reviewer 2: ```suggestion
To take full advantage of the title formatting functionality, set the right attribute to be used as the main product title
```

# Where to see data quality grades?

> Data quality grades for **enrichment** and **consistency** are displayed as badges on various screens of the PIM.

Reviewer 2: ```suggestion
Data quality grades for **enrichment** and **consistency** are displayed as badges on various PIM screens.
```

## On your dashboard

> Your **dashboard** offers multiple ways of measuring your catalog's enrichment and consistency. You can choose to view data quality on the whole catalog, on a family or on a category.

Reviewer: ```suggestion
Your **dashboard** offers multiple ways of measuring your catalog's enrichment and consistency. You can choose to view data quality on the whole catalog, on a family, or on a category.
```

![Image data quality dashboard](../img/data-quality-dashboard.png)

## In your product grid
Your **product grid** will display the enrichment and consistency grades for each product.

![Image product grid](../img/data-quality-grid.png)

## On your product edit form
Your **product edit form** will display the enrichment and consistency grades for each product.

![Image product edit form](../img/data-quality-pef.png)

::: info

> Enrichment and consistency grades are not yet calculated on product variant and product models. Stay tuned for our future releases!

Reviewer 2: ```suggestion
Enrichment and consistency grades are not yet calculated on variant products and product models. Stay tuned for our future releases!
```

:::

## In your data quality insights panel (product edit form)

> Clicking on `data quality insights` in the product edit form leads you to a panel dedicated so that you understand your grades.

Reviewer: ```suggestion
Clicking on `data quality insights` in the product edit form leads you to a dedicated panel so that you understand your grades.
```
